daemon/daemon_main.go

```diff
@@ -29,7 +29,6 @@ import (
 	"github.com/cilium/cilium/api/v1/server"
 	"github.com/cilium/cilium/api/v1/server/restapi"
 	"github.com/cilium/cilium/common"
-	_ "github.com/cilium/cilium/pkg/alignchecker"
```

Comment:
The commit message about side-effects. Is that because of this PR or it was just true in general?

Reply:
It was true in general even before this PR. That commit slipped into the PR by accident, I intended to send it separately. Sorry about that.